packages/integrations/node/src/http-server.ts

```diff
@@ -23,21 +23,36 @@ function parsePathname(pathname: string, host: string | undefined, port: number)
 
 export function createServer(
 	{ client, port, host, removeBase }: CreateServerOptions,
-	handler: http.RequestListener
+	handler: http.RequestListener,
+	trailingSlash: string
```

Comment:
We lose type information here. We should use `AstroUserConfig['trailingSlash']` here, so we use the same type of the configuration